topics/ecology/tutorials/lotus2-identifying-fungi/tutorial.md

```diff
@@ -0,0 +1,462 @@
+---
```

Comment:
Question: This tutorial is very similar to many tutorials in the microbiome topic in terms of tools and techniques used, do you think it makes sense to move it there? And maybe we can then add a tag 'ecology' orso?

Reply:
Ok. Will move there and add
```
tags:
- fungi
- ecology
```

Reply:
Hi @shiltemann @paulzierep @hexylena 

Thank you so much for your time and the excellent comments and suggestions. I really really appreciate them as I'm new to writing galaxy tutorials. Your time and energy this first time will hopefully mean very light editing in future tutorials :-) 

One potential problem with moving the whole tutorial folder to topics/microbiome is that I think the link between these comments and the tutorial text in the original folder gets lost? Hopefully I'm wrong about that.

But if so, I can reassure you that I have done the changes that I have resolved in these comments! :-)

Am about to submit a new commit to this pull request with all the changes but just wanted to give you a heads up.